projects/packages/my-jetpack/src/class-wpcom-products.php

```diff
@@ -35,20 +36,34 @@ class Wpcom_Products {
 	 * @return Object|WP_Error
 	 */
 	private static function get_products_from_wpcom() {
+		$connection        = new Connection_Manager();
+		$is_user_connected = $connection->is_user_connected();
+
+		// Request parameters.
+		$params = array(
+			'method'  => 'GET',
+			'headers' => array(
+				'X-Forwarded-For' => ( new Visitor() )->get_ip( true ),
+			),
+		);
+
+		$endpoint = sprintf( '/products/?_locale=%s&type=jetpack', get_user_locale() );
```

Comment:
We need to use `/sites/$site/products` endpoint because it provides relevant data to the site :-/